docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
```

Comment:
Please can you follow the existing format for MTA - https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.1/html/release_notes/mta-6-1-0#rn-known-issues-610_release-notes

![image](https://github.com/windup/windup-documentation/assets/106804821/a451e9ec-ef38-4843-93b4-f518883d5983)